scalableminds/webknossos
# Tag buildinfo

### Steps to test:
- start webKnossos with `CIRCLE_TAG=18.07.0 ./sbt run` or `CIRCLE_BUILD_NUM=1234 ./sbt run`  and check the help menu in the navbar

### Issues:
- fixes #2887 

------
